### PR TITLE
Have `filepicker` accept default observables

### PR DESCRIPTION
**Pull request by ranjanan**
Firstly, thank you for such a useful package! This package greatly simplifies writing Web-based UIs . 😄

I was hoping to make a simple contribution. This PR modifies `filepicker` to take in a default observable, much like `checkbox`, for example.

### CODE REVIEW

**Comment by piever** [comment]
Thanks! This was something I had forgotten to do. I left a few comments, in particular:

- make sure tests pass
- make sure it works for any sensible initial value the user may give (`nothing`, a string or a `Observable`).

**Comment by piever** [comment]
I actually like the idea of having the observable start at `nothing` and in the `@manipulate` the `@manipulate for x = nothing` could create a file selection widget. Say:

```julia
@manipulate for x = nothing
  x != nothing && histogram(column(loadtable(x), 1))
end
```

File: src/input.jl [hunk 1]
```diff
@@ -6,7 +6,7 @@ If `multiple=true` the observable will hold an array containing the paths of all
 selected files. Use `accept` to only accept some formats, e.g. `accept=".csv"`
 """
 function filepicker(::WidgetTheme, lbl="Choose a file...";
-    label=lbl, class="", multiple=false, kwargs...)
+    label=lbl, class="", value = nothing, multiple=false, kwargs...)
```
[the remark this answers is not in the document]
Comment (ranjanan) [reply src/input.jl]: Sure, makes sense. I've added that.

**Comment by piever** [comment]
Nice job!

**Pull request merged**